content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+  - [Single Handler](#single-handler)
+      - [Single Handler with a Single Matching Contact](#single-handler-with-a-single-matching-contact)
+      - [Single Handler with a Single Non-Matching Contact](#single-handler-a-single-non-matching-contact)
+      - [Single Handler with Multiple Matching Contacts](#single-handler-with-multiple-matching-contacts)
+      - [Single Handler with Multiple Non-Matching Contacts](#single-handler-with-multiple-non-matching-contacts)
+      - [Single Handler with Some Matching Contacts](#single-handler-with-some-matching-contacts)
+  - [Multiple Handlers](#multiple-handlers)
+      - [Multiple Handlers with a Matching Contact](#multiple-handlers-with-a-matching-contact)
+      - [Multiple Handlers with a Single Matching Contact](#multiple-handlers-with-a-single-matching-contact)
+      - [Multiple Handlers with a Single Non-Matching Contact](#multiple-handlers-with-a-single-non-matching-contact)
+      - [Multiple Handlers with Multiple Matching Contacts](#multiple-handlers-with-multiple-matching-contacts)
+      - [Multiple Handlers with Multiple Non-Matching Contacts](#multiple-handlers-with-multiple-non-matching-contacts)
+      - [Multiple Handlers With Some Matching Contacts](#multiple-handlers-some-matching-contacts)
+- [Wrapping Up](#wrapping-up)
+  - [Contact Routing and Sensu Event Pipeline](#contact-routing-and-sensu-event-pipeline)
+- [References](#references)
+- [Additional Resources](#additional-resources)
+
+# Prerequisites
+
+Before diving into this guide, we recommend having the following components ready:
+
+- A working Sensu Enterprise deployment
+- Two or more handlers configured
+
+If you've not already signed up for Sensu Enterprise, you can do so via [this link][1].
+
+In this guide we'll be using [Slack][2] and [Email][3] handlers for our demonstration.
+
+# Overview
+
+Every incident, outage or event has an ideal first responder. This can be either
+a team or individual with the knowledge to tirage and address the issue. Sensu
+Enterprise contact routing makes it possible to assign checks to specific teams
+and/or individuals, reducing mean time to response and recovery (MTTR). Contact
+routing works with all of the Sensu Enterprise third-party notification and
+metric integrations.
+
+_NOTE: Sensu Enterprise supports contact routing for all integrations except
+EC2, Puppet, Chef, Flapjack, and Event Stream._
+
+In this guide we'll cover configuring and using Sensu Enterprise Contact
+
+## Contact Routing Basics
+
+In Sensu Enterprise Contact Routing, contacts are composed of a name and
+configuration overrides for one or more of Sensu Enterprise's built-in
+integrations or handler configurations. A contact in Sensu Enterprise is not too
+dissimilar from a contact on your phone, which usually have a name and one or
+more _identifiers_ for various communication **channels** (e.g. a **phone**
+_number_, **email** _address_, **Twitter** _username_, etc).
+
+### Contact Configuration
```

Comment:
Might be worth it to show a handler somewhere near here, as I could see a user thinking this is all the need for contact routing to work, but you'll still need a handler definition as well.

Reply:
added handler and integration examples